docs/source/tts/configs.rst

```diff
@@ -96,7 +96,6 @@ Text normalization (TN) converts text from written form into its verbalized form
       _target_: nemo_text_processing.text_normalization.normalize.Normalizer
       lang: en
       input_case: cased
-      whitelist: "nemo_text_processing/text_normalization/en/data/whitelist/lj_speech.tsv"
```

Comment:
As discussed offline: please add a paragraph in the documentation describing how default/non-default settings for `whitelist` work in a future PR.